src/CrowdedMod/CrowdedMod.csproj

```diff
@@ -21,6 +21,9 @@
         <PackageReference Include="BepInEx.AutoPlugin" Version="1.1.0" PrivateAssets="all" />
         <PackageReference Include="BepInEx.IL2CPP.MSBuild" Version="2.0.1" PrivateAssets="all" />
     </ItemGroup>
+    <ItemGroup>
+      <PackageReference Update="AmongUs.GameLibs.Steam" Version="2024.6.18" />
+    </ItemGroup>
```

Comment:
AmongUs libs are already included, you should change the version in project properties

Reply:
my ide told to do this